spec/whois/record/parser/blank_spec.rb

```diff
@@ -8,7 +8,7 @@
   }
 
   Whois::Record::Parser::PROPERTIES.each do |method|
-    describe method do
+    describe "##{method}" do
```

Comment:
`Whois::Record::Parser::PROPERTIES` is a list of symbols and when a symbol is passed to `describe`, RSpec complains:

```
The semantics of `describe <a Symbol>` are changing in RSpec 3. In RSpec 2,
this would be treated as metadata, but as the first `describe` argument,
this will be treated as the described object (affecting the value of
`described_class`) in RSpec 3. If you want this to be treated as metadata,
pass a description as the first argument.
(Example group defined at .../whois/spec/whois/record/parser/blank_spec.rb:11:in `block (2 levels) in <top (required)>')
```

So we could go with `describe method.to_s` but since it's an instance method I find `describe '#foo'` nicer than `describe 'foo'`.

Your call! :smile_cat: